app/controllers/carl_people_controller.rb

```diff
@@ -0,0 +1,47 @@
+class CarlPeopleController < ApplicationController
+  #----------------- SHOW A PERSON --------------------
+  def show
+    @person = Person.find(params[:id])
+    @tasks_count = @person.tasks.count
+    @tasks_wip = @person.tasks.where(date_complete: nil).count
+    @tasks_done = @tasks_count - @tasks_wip
+  rescue
+    redirect_to "/person_not_found"
```

Comment:
It looks like you are adding a custom 404 page - you can actually do this just by updating the public/404.html page which will display automatically when the route isn't recognized or when a controller raises a `ActiveRecord::RecordNotFound` exception. That page has to be static, but there are several more complicated ways to serve dynamic 404 pages in rails, I'd just google around for a tutorial if you're curious.